grumphp.yml

```diff
@@ -0,0 +1,80 @@
+parameters:
+    bin_dir: "./vendor/bin"
+    git_dir: "."
+    hide_circumvention_tip: false
+    hooks_dir: ~
+    hooks_preset: local
+    ignore_unstaged_changes: false
+    process_async_limit: 10
+    process_async_wait: 1000
+    process_timeout: 60
+    stop_on_failure: true
+    ascii:
+        failed: ~
+        succeeded: ~
+    tasks:
+        composer:
+            file: ./composer.json
+            no_check_all: false
+            no_check_lock: false
+            no_check_publish: false
+            with_dependencies: false
+            strict: true
+        git_blacklist:
+            keywords:
+                - "die("
+                - "dump("
+                - "exit;"
+                - "print_r("
+                - "print_rr("
+                - "var_dump("
+            triggered_by: ['php']
+        phpcpd:
+            directory: '.'
+            exclude: ['vendor']
+            names_exclude: []
+            fuzzy: false
+            min_lines: 5
+            min_tokens: 70
+            triggered_by: ['php']
+        phpcsfixer2:
+            allow_risky: false
+            cache_file: ~
+            config: ~
+            rules: []
+            using_cache: true
+            path_mode: ~
+            verbose: true
+        phplint:
+            exclude: ['vendor']
+            jobs: ~
+            triggered_by: ['php', 'phtml']
+        phpstan:
```

Comment:
Does GrumPhp trigger PhpStan when used with PHP 5.6?

We currently still support both 5.6 and 7.0, PhpStan is a dev dependency which doesn't _need_ to be run.